internal/handlers/telegram/hander_test.go

```diff
@@ -0,0 +1,60 @@
+package telegram_test
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+	"github.com/kyokomi/emoji"
+	"github.com/ritlug/teleirc/internal"
+	bridge "github.com/ritlug/teleirc/internal/handlers/telegram"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestStickerSmile(t *testing.T) {
+	correct := "<test>😄"
+	updateObj := tgbotapi.Update{
+		Message: &tgbotapi.Message{
+			From: &tgbotapi.User{
+				UserName: "test",
+			},
+			Sticker: &tgbotapi.Sticker{
+				Emoji: emoji.Sprint(":smile:"),
+			},
+		},
+	}
+
+	clientObj := &bridge.Client{}
+
+	clientObj.Settings = internal.TelegramSettings{
+		Prefix: "<",
+		Suffix: ">",
+	}
+	clientObj.SendToIrc = func(s string) {
+		assert.Equal(t, correct, s)
+	}
+
+	// clientObj.handler.stickerHandler(clientObj, updateObj)
```

Comment:
I need some help. Turning the test file to telegram_test, though I agree this is the best course of action, I cant call the sticketHandler directly. 

What @Zedjones did with mocking the client in testing IRC is a better structure, so I will attempt to follow suite. However, his project isnt in the irc_test namespace either. 

Can someone help me get a better grasp of how to handle this?